connect_your_app_to_IRIS/technical_details/app_onboarding.md

```diff
@@ -27,13 +27,26 @@ Please find below and overview of the IRIS system and its actors as well as the
 ## Request a certificate
 
 ToDo: Describe process how to contact the rollout team and what information is expected in the CSR. 
-## Install and configure EPS
 
-First, the EPS is installed and configured locally. Then messages can be sent to and received from IRIS via the EPS.
+Generate your certificate signing request 
 
-The EPS server can be cloned and created from [https://github.com/iris-gateway/eps](https://github.com/iris-gateway/eps). We will later also provide an executable, versioned binary for all common environments.
+    openssl genrsa -out "${cert}.key" 2048;
+  	openssl rsa -in "${cert}.key" -pubout -out "${cert}.pub";
+  	openssl req -new -sha256 -key "${cert}.key" -subj "/C=${C}/ST=${ST}/L=${L}/O=${O}/OU=${OU}/CN=${cert}" -addext "subjectAltName = DNS:${cert},DNS:*.${cert}.local" -out "${cert}.csr";
 
+with your information in subject and your app name as ${cert}. Please use your app name as CN (for example CN=smartmeeting). Don't use spaces. 
```

Comment:
I suggest to provide a full example from Smart Meeting. That will make it really clear what we expect. 

```
O="COSYNUS GmbH"
ST="Europaplatz 5"
L="64293 Darmstadt"
C="DE"
OU="IT"
CN="smartmeeting"
# using less than 1024 here will result in a TLS handshake failure in Go
# using less than 2048 will cause e.g. 'curl' to complain that the ciper is too weak
LEN="2048"
```